bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/password.js

```diff
@@ -174,6 +174,12 @@ pimcore.object.classes.data.password = Class.create(pimcore.object.classes.data.
                 hideLabel: true,
                 value: t('width_explanation')
             },
+            {
+                xtype: "numberfield",
+                fieldLabel: t("min_length"),
+                name: "minimumLength",
+                value: this.datax.minimumLength
```

Comment:
Nice. Could you please add this to the config (just to avoid to get into the negative range)

minValue: 0